dbms/tests/queries/0_stateless/00800_function_java_hash.sql

```diff
@@ -0,0 +1,4 @@
+select JavaHash('abc');
```

Comment:
Better to have names starting with lowercase and with number of bits (`javaHash32`).

Reply:
ok